fredericoo/saltplay
# feat: implement multiple players per team capability on the database

# Overview

This PR aims to refactor some code to enable multiple players per team on the database.

## What we have done

- Slightly revamped the player picker to better accomodate mutiple selections, make the selection more prominent (some people were dubious as to whether their choice was selected or just hovered) and redesigned the search with a smaller field with an icon to make it even clearer
- Migrated the database with the creation of supporting tables to hold team relations in postgres
- Replaced names and functionality across the entire app to the new API

## How to test

- Run the staging version on Vercel, check whether the New Match design has changed;
- See the current matches on some games; they should be visible and working normally (bear in mind that it's querying from  `_left`  and `_right`, which makes it very tricky!
- Try submitting a match, it should work correctly.

## Checklist:
